adding PHP WF name (#1719)
temporalio/documentation: docs-src/php/how-to-customize-workflow-type-in-php.md

---
id: how-to-customize-workflow-type-in-php

> title: How to customize Workflow types in PHP

Reviewer: ```suggestion
title: How to customize a Workflow Type in PHP
```

> sidebar_label: Customize Workflow types

Reviewer: ```suggestion
sidebar_label: Custom Workflow Type
```

> description: Customize Workflow types.

Reviewer: ```suggestion
description: How to customize a Workflow Type in PHP
```

> description: Customize Workflow types.

Reviewer 2: ```suggestion
description: To customize a Workflow type, use the Workflow method annotations to specify the name of Workflow.
```

tags:
  - developer-guide
  - sdk
  - php
---

> To customize a Workflow type, use the Workflow method annotations to specify the name of Workflow.

Reviewer: ```suggestion
To customize a Workflow Type, use the `WorkflowMethod` annotation to specify the name of Workflow.
```

```php
#[WorkflowMethod(name)]
```

> If a Workflow type is not specified, then Workflow type defaults to the interface name, `FileProcessingWorkflow`.

Reviewer 2: ```suggestion
If not specified, Workflow type defaults to the interface name, `FileProcessingWorkflow`.
```

> If a Workflow type is not specified, then Workflow type defaults to the interface name, `FileProcessingWorkflow`.

Reviewer: ```suggestion
If a Workflow Type is not specified, then Workflow Type defaults to the interface name, which is `FileProcessingWorkflow` in this case.
```

```php
#[WorkflowInterface]

> interface FileProcessingWorkflow

Reviewer: We should make this YourWorkflowDefinitionInterface
